beesdoo_pos/static/src/js/beesdoo.js

```diff
@@ -91,23 +77,8 @@ odoo.define("beesdoo_pos.screens", function (require) {
                         ".customer-name",
                         self.pos.get_client().name
                     );
-                    set_customer_info.call(
-                        self,
-                        ".customer-delegate1",
-                        result[0],
-                        "Eater 1: "
-                    );
-                    set_customer_info.call(
-                        self,
-                        ".customer-delegate2",
-                        result[1],
-                        "Eater 2: "
-                    );
-                    set_customer_info.call(
-                        self,
-                        ".customer-delegate3",
-                        result[2],
-                        "Eater 3: "
+                    result.forEach((element) =>
+                        set_customer_info.call(self, ".customer-delegates", element)
```

Comment:
i wouldn’t call the variable `element` because it implies that it is an html element, while it is the name of person (as a string), if i understand correctly. `customer_name`, maybe?

didn’t you lost the “Eater <n>: ” prefix? is this wanted?

Reply:
- OK, I went for `client_name` instead, to be consistent with the `get_client` method.

- Yes I lost the "Eater 1:" prefix, I am waiting for the analyst's answer to see if it's OK or what would be a better output.